dialog_select.go

```diff
@@ -54,6 +54,20 @@ func NewStaticSelectDialogInput(name, label string, options []DialogSelectOption
 	}
 }
 
+// NewDynamicSelectDialogInput constructor for a `external` datasource menu input
+func NewDynamicSelectDialogInput(name, label string, options []DialogSelectOption) *DialogInputSelect {
```

Comment:
Prefer to use `External` instead of `Dynamic`.

Reply:
@kanata2 thank you for suggestion, fixed